python/ffsim/qiskit/sim.py

```diff
@@ -379,6 +380,19 @@ def _evolve_state_vector_spinless(
         vec = _apply_iswap(vec, (i, j), norb=norb, nelec=nelec, copy=False)
         return states.StateVector(vec=vec, norb=norb, nelec=nelec)
 
+    if isinstance(op, PermutationGate):
+        perm = list(op.pattern)
+        qs = qubit_indices.copy()
+        for k in range(len(perm)):
```

Comment:
I think this can be improved by factoring out a generator function that yields the swap pairs of the permutation decomposition. Then this can look something like
```python
for i, j in _decompose_permutation(perm):
    vec = _apply_swap(vec, (qs[i], qs[j]), ... )